util/usernames

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
with hashbang + `chmod +x`, it's not necessary unless you're trying to use it as a python library. I like it this way because I don't like it when command line utilities (like this one) expose what language they're written in via their filename. Agree/disagree?